src/main/java/com/nooul/apihelpers/springbootrest/controllerAdvices/BodyAdvice.java

```diff
@@ -1,49 +0,0 @@
-package com.nooul.apihelpers.springbootrest.controllerAdvices;
```

Comment:
I definitely need that I will readd it - I don't want to be returning pure arrays that are not in { content: [] } is it happens already when there is pagination.

Reply:
Got it.